tests/php/test-class-amp-dom-document.php

```diff
@@ -10,7 +10,7 @@
 /**
  * Tests for Amp\AmpWP\Dom\Document.
  *
- * @covers Document
+ * @covers Amp\AmpWP\Dom\Document
```

Comment:
Does CodeCov not recognize imports?

Reply:
It not sure if it does, but PHPUnit requires the full namespace or else it [fails to find the class](https://travis-ci.org/ampproject/amp-wp/jobs/650997147#L1068-L1069). I thought it was fixed via https://github.com/sebastianbergmann/phpunit/issues/1898, but it may be because an older version of PHPUnit without the fix is being used.